docs/documentation.md

```diff
@@ -0,0 +1,50 @@
+<frontmatter>
+  title: "Getting Started: Documentation"
+</frontmatter>
+
+# MarkBind Documentation
+
+This project uses [MarkBind](https://markbind.org/) for developer documentation.
+
+## Quickstart
+
+MarkBind is used to create a static site for developer documentation, and can parse markdown, git flavoured markdown and more. Generally, edits to documentation files in docs will be reflected on the website with no further action, due to auto-deployment by Github actions.
+
+To preview changes to documentation on the markbind:
+
+1. Install `npm` (version 7 or higher) and `Node.js` (version 12 or higher)
+1. Navigate to the `/docs` folder in your fork of the TEAMMATES repository
+1. Run `npm i` to install the necessary tools to build documentation, such as MarkBind.
```

Comment:
> `npm ci`, not `npm i`

Hi @wkurniawan07, there is no reason why it should be `npm ci` and not `npm i`.

Reply:
@tlylt
- `npm i` opens up the risk of using a version of MarkBind (or any transitive dependency) different from defined in the lockfile, and running into errors due to different version.
- If `npm i` writes over the `package-lock.json`, it either troubles the dev because s/he has to revert the file before staging any change, or the reviewer because s/he may see unnecessary change in the file.

You really don't need `npm i` other than for adding new dependencies or updating dependencies beyond its version specifier.

Reply:
@wkurniawan07

> You really don't need npm i other than for adding new dependencies or updating dependencies beyond its version specifier.

It's interesting how you decided to use `npm ci` in a context outside of what it is meant to [do](https://docs.npmjs.com/cli/v8/commands/npm-ci). I don't think this is the standard approach in typical nodejs projects but I understand your decision better now. Thanks for the explanation.

Reply:
> Install a project _with a clean slate_

> it's meant to be used in automated environments such as test platforms, continuous integration, and deployment -- or _any situation where you want to make sure you're doing a clean install of your dependencies_.

In the context of a fresh setup of a project, I dare say `npm ci` is the one that fits the bill.

Of course, the risk associated with `npm i` is rather small (if not tiny), but why subjecting ourselves to one if there is an alternative that presents zero?